src/libraries/System.Transactions.Local/ref/System.Transactions.Local.cs

```diff
@@ -191,6 +191,7 @@ public static partial class TransactionManager
         [System.Diagnostics.CodeAnalysis.DisallowNullAttribute]
         public static System.Transactions.HostCurrentTransactionCallback? HostCurrentCallback { get { throw null; } set { } }
         public static System.TimeSpan MaximumTimeout { get { throw null; } set { } }
+        public static bool ImplicitDistributedTransactions { get; set; }
```

Comment:
cc: @bartonjs, @terrajobst for the new public API

Reply:
Being discussed in #76469